torch_xla/csrc/ops/einsum.cpp

```diff
@@ -0,0 +1,69 @@
+#include "torch_xla/csrc/ops/einsum.h"
+
+#include "tensorflow/compiler/xla/client/lib/matrix.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+
+namespace torch_xla {
+namespace {
+
+xla::XlaOp BuildEinsum(absl::Span<const xla::XlaOp> operands,
```

Comment:
can you move this to https://github.com/pytorch/xla/blob/master/torch_xla/csrc/reduction.h ?

Reply:
Done